airflow/www/templates/airflow/dag.html

```diff
@@ -28,13 +28,13 @@
 <div>
     <h3 class="pull-left">
       {% if dag.parent_dag is defined and dag.parent_dag %}
-        <span style='color:#AAA;'>SUBDAG: </span> <span> {{ dag.dag_id }}</span>
+        <span class="text-gray">SUBDAG: </span> <span> {{ dag.dag_id }}</span>
       {% else %}
         <input id="pause_resume" dag_id="{{ dag.dag_id }}" type="checkbox" {{ "checked" if not dag.is_paused else "" }} data-toggle="toggle" data-size="mini" method="post">
-        <span style='color:#AAA;'>DAG: </span> <span> {{ dag.dag_id }}</span> <small class="text-muted"> {{ dag.description }} </small>
+        <span class="text-gray">DAG: </span> <span> {{ dag.dag_id }}</span> <small class="text-muted"> {{ dag.description }} </small>
       {% endif %}
       {% if root %}
-        <span style='color:#AAA;'>ROOT: </span> <span> {{ root }}</span>
+        <span class="text-gray">ROOT: </span> <span> {{ root }}</span>
```

Comment:
Can we come up with a different name for this class - I was taught that having a property of the visual style in the class name is a code smell. `class="dag-location"`? `class="dag-info-label"?

Reply:
Whatever  class name we pick should work for task_instance.html too.

One option might be to change the HTML a bit too:

```
<h3><span>ROOT:</span> {{ root }}</h3>
```

And then 

```
h3 span {
color: #AAA;
}
```

We might want to apply a class on the H3, or on the first span (and remove the second span?)


```
<h3><span class="type">ROOT:</span> {{ root }}</h3>
```

Reply:
`h3 span` looks interesting to me. I will look at how I can change the name of the class.